cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -283,6 +293,66 @@ public Future<Void> reconcile(KafkaStatus kafkaStatus, Clock clock)    {
                 .compose(i -> updateKafkaMetadataState(kafkaStatus));
     }
 
+    private Future<Void> updateKafkaAutoRebalanceStatus(KafkaStatus kafkaStatus) {
+        // we are going to save a consistent list of added notes (between the ones already in the status and the newly added ones)
+        // into the KafkaStatus autorebalance field to be passed to the auto-rebalancing reconciler.
+        // It's needed in case nodes are scaled up but reconciliation fails and on the next one the operator has to know where to rebalance
+
+        Set<Integer> newAddedBrokers = new LinkedHashSet<>(kafka.addedBrokerNodes());
```

Comment:
So ... new? Or added? Maybe one would be enough.

Reply:
LOL. Agree ;-)